ambuild2/context.py

```diff
@@ -92,17 +93,34 @@ def reconfigure(self):
 
         # The database should be upgraded here, so we should always have an
         # API version set.
-        api_version = self.db.query_var('api_version')
+        api_version = Version(self.db.query_var('api_version'))
         assert api_version is not None
 
-        if api_version == '2.0':
-            from ambuild2.frontend.v2_0.amb2.gen import Generator
-        elif api_version == '2.1':
-            from ambuild2.frontend.v2_1.amb2 import Generator
-
-        gen = Generator.FromVars(self.vars, self.db, self.options.refactor)
+        from ambuild2.frontend.amb2 import Generator
+
+        if api_version >= '2.1':
+            from ambuild2.frontend.v2_1.context_manager import ContextManager
+        elif api_version >= '2.0':
+            from ambuild2.frontend.v2_0.context_manager import ContextManager
```

Comment:
I have a better import suggestion:
```suggestion
        ContextManager = __import__('ambuild2.frontend.v{0}.context_manager'.format(api_version.replace('.', '_')),
                                    fromlist=('*',)).ContextManager
```
Idk if it's format correct though.